lib/xenops_server.ml

```diff
@@ -96,7 +96,7 @@ type atomic =
 	| VM_shutdown_domain of (Vm.id * shutdown_request * float)
 	| VM_s3suspend of Vm.id
 	| VM_s3resume of Vm.id
-	| VM_save of (Vm.id * flag list * data)
+	| VM_save of (Vm.id * flag list * data * data option)
```

Comment:
Is it worth a comment describing the uses/meanings of the elements in this four-tuple anonymous type?
(Using a record with named fields is probably over the top.)